client/src/Landing/Landing.jsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import './css/Landing.css';
+import CorndelLogo from './img/Corndel-Logo.png'
+import placeholders from './usernames.js';
+
+export default class Landing extends React.Component {
+	constructor(props){
+		super(props)
+		this.state = {username: ""}
+	}
+
+	componentDidMount(){
+		let ranName = placeholders[Math.floor(Math.random() * placeholders.length)]
+		document.getElementById('username').placeholder = ranName.name;
+	}
+
+	validateForm(){
+		return this.state.username.length > 0;
+	}
+
+	handleSubmit(){
+		fetch('/', {
+			method: "POST",
+			headers: { "Content-Type": "application/json" },
+			body: JSON.stringify(this.state)
```

Comment:
I think this is fine for now but in general we shouldn't just send the entire `state`, only what we want to.